src/core/public/saved_objects/simple_saved_object.ts

```diff
@@ -77,9 +80,14 @@ export class SimpleSavedObject<T = unknown> {
 
   public save(): Promise<SimpleSavedObject<T>> {
     if (this.id) {
-      return this.client.update(this.type, this.id, this.attributes, {
-        references: this.references,
-      });
+      return this.client
+        .update(this.type, this.id, this.attributes, {
+          references: this.references,
+        })
+        .then((sso) => {
+          this.updatedAt = sso.updatedAt;
+          return sso;
+        });
```

Comment:
wasn't sure if this change warrants adding a test. thoughts?

Reply:
I know that the current test suite is pretty lackluster, but let's not make the situation worse, and let's add tests on new enhancements if that's fine with you.

Reply:
happy to help. don't mind iterating too if needed 👍